examples/algo-example/src/index.ts

```diff
@@ -15,27 +15,36 @@ import { SDKArguments } from '@tatumio/shared-abstract-sdk'
 
 export const sdkArguments: SDKArguments = {
   apiKey: '75ea3138-d0a1-47df-932e-acb3ee807dab',
+  url: 'http://127.0.0.1:3000',
   // url: 'https://api.tatum.io',
+  provider: '',
```

Comment:
Delete this line. On above line, use https://api.Tatum.io/v3/blockchain/node/ALGO